src/http_rest_api_handler.cpp

```diff
@@ -838,11 +879,11 @@ Status HttpRestApiHandler::processRequest(
 
     headers->clear();
     response->clear();
-    headers->push_back({"Content-Type", "application/json"});
+    (*headers)["content-type"] = "application/json";
```

Comment:
Should this line still be here since we can now support other content types and here we do hard assignment.

Reply:
_very_ good catch. all our tests use dispatchToProcessor directly, therefore we never noticed this overwrite! will remove it

Reply:
fixed